components/clarinet-files/src/lib.rs

```diff
@@ -242,6 +213,59 @@ impl FileLocation {
         }
     }
 
+    pub fn get_project_root_location(&self) -> Result<FileLocation, String> {
```

Comment:
This method has been moved closer to "get_root_location_from_manifest_location" but unchanged